libs/shared/src/types/auth/auth.types.ts

```diff
@@ -11,6 +13,18 @@ export interface IJwtClaims {
   iss?: string;
 }
 
+// JWT payload + custom claims
+export type ClerkJwtPayload = JwtPayload & {
```

Comment:
This type definition is a bit suspicious. We have the community and the Clerk JWT payload I'd expect to have either a common interface or transform one to the other. At this point, I'd suggest making the clerk JWT the source of truth and adopt the rest accordingly.

Reply:
This type will eventually be not needed here - after we implement the JWT env switch it will live only in `clerk.strategy.ts` and not in shared package. 

The only 2 places where this type is used is:
-  `clerk.strategy.ts -> validate(payload: ClerkJwtPayload)` - its what the validate function gets right after decoding the JWT so its a correct type here, can't be transformed before this
- `user.session.ts` -> updateEETokenClaims()` - which is used only for testing when switching tokens and will be eventually removed

Reply:
Awesome.